compaction/compaction_manager.cc

```diff
@@ -622,7 +622,7 @@ void compaction_manager::submit(column_family* cf) {
     });
 }
 
-void compaction_manager::submit_offstrategy(column_family* cf) {
+future<> compaction_manager::submit_offstrategy(column_family* cf) {
```

Comment:
i think we should keep this variant which doesn't return a future.

```
private:
lw_shared_ptr<task> do_submit_offstrategy(column_family*);
public:
future<> perform_offstrategy(column_family*) { 
  auto task = do_submit_offstrategy(cf);
  return task->compaction_done.get_future().finally([task] {});
}
void submit_offstrategy(column_family*) {
  do_submit_offstrategy(cf);
}
```

Reply:
Fixed in v3